x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -49,3 +81,209 @@ export const createConfigurationSubClient = (
 
   return Object.freeze(configureSubClient);
 };
+
+export const createConfigurationSubClient = (
```

Comment:
I placed all the configuration APIs in a single file because they're fairly small.